File: Server/jade/src/main/java/tech/hirsun/jade/pojo/Picture.java
package tech.hirsun.jade.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Picture {
    private Integer id;
    private Integer userId;
    private Integer topicId;
    private String fileName;

    private Integer viewCount;
    private String location;
    private String coordinateX;
    private String coordinateY;

    private String title;
    private String description;
    private Date createTime;

    private String exifSize;
    private String exifTime;
    private String exifLatitude;
    private String exifLongitude;
    private String exifLocation;
    private String exifDevice;
}

File: FrontEnd/Jade/app/src/main/java/com/iems5722/jade/pojo/Picture.kt
package com.iems5722.jade.pojo

import java.util.Date

data class Picture(
    val id: Int,
    val userId: Int,
    val topicId: Int,
    val fileName: String,
    val viewCount: Int,
    val location: String,
    val coordinateX: String,
    val coordinateY: String,
    val title: String,
    val description: String,
    val createTime: Date,
    val exifSize: String,
    val exifTime: String,
    val exifLatitude: String,
    val exifLongitude: String,
    val exifLocation: String,
    val exifDevice: String
)
